atlasdb-jepsen-tests/src/test/java/com/palantir/atlasdb/jepsen/events/EventTest.java

```diff
@@ -94,18 +95,75 @@ public void canDeserialiseInvokeEvent() {
     }
 
     @Test
-    public void canDeserialiseOkRead() {
+    public void canDeserialiseOkReadWithValueAsStringOfLong() {
         Map<Keyword, Object> keywordMap = new HashMap<>();
         keywordMap.put(Keyword.intern("type"), Keyword.intern("ok"));
         keywordMap.put(Keyword.intern("f"), Keyword.intern("read-operation"));
-        keywordMap.put(Keyword.intern("value"), SOME_VALUE);
+        keywordMap.put(Keyword.intern("value"), SOME_INTEGER_AS_STRING);
```

Comment:
nit: This doesn't match up with the test name. Maybe `SOME_LONG_AS_STRING` would be better, or change the test name?